receiver/vcenterreceiver/metadata.yaml

```diff
@@ -334,6 +349,39 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.resource_pool.memory.swapped:
+    enabled: false
+    description: The amount of memory that is granted to VMs from the host's swap space.
```

Comment:
@BominRahmani A smaller thing and won't hold up the PR for this but...: Maybe it would be a little less confusing for a user to read about if all of these descriptions referenced resource pool. For example "The amount of memory that is granted to VMs in the resource pool from the host's swap space."